examples/bigcommerce/lib/bigcommerce/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { getProduct, getProducts } from './bigcommerce'
+
+export const getProductSlugPaths = async (): Promise<string[]> => {
+  const products = await getProducts()
+
+  return products?.map((edge: any) => edge.node.entityId)?.map((entityId: any) => entityId) ?? []
+}
+
+export const getProductBySlug = async (slug: string): Promise<number> => {
+  return await getProduct(parseInt(slug))
```

Comment:
Doesn't the `getProduct` function return a `Promise<Product>` and not a `Promise<number>`?